src/less/flex.less

```diff
@@ -60,7 +60,7 @@
  * @param {number} basis - See `.flex-basis`. Defaults to auto.
  * @initial {0 1 auto}
  */
-.flex(@grow, @shrink: 1, @basis: auto) {
+.flex(@grow: 0, @shrink: 1, @basis: auto) {
```

Comment:
not sure it's useful to call `.flex()` with empty default arguments, since that's the browser initial value to begin with. probably better for people to be explicit about grow/shrink properties